Archivist/LODwAPArchivist/LODwAPArchivist.cpp

```diff
@@ -204,16 +202,25 @@ bool LODwAPArchivist::archive(std::vector<std::shared_ptr<Organism>> &population
 
   // Save Data
   if (writeDataFile) {
-    auto time_to_coalescence = writeLODDataFile(LOD, real_MRCA, effective_MRCA);
-    if (flush) {
-      if (real_MRCA->timeOfBirth != -2) {
-	std::cout << "Time to Coalescence for last saved organism on LOD is "
-	<< time_to_coalescence << " updates ago." << std::endl;
-      }
-      else {
-	std::cout << "This run has not coalesced. There is no Most Recent Common Ancestor." << std::endl;
-      }
-    }
+	  writeLODDataFile(LOD, real_MRCA, effective_MRCA);
+	  if (flush) {
+		  if (real_MRCA->timeOfBirth == -2) {
+			  std::cout << "This run has not coalesced. There is no Most Recent Common Ancestor.\n" <<
+				  "None of the organisms in LOD_data.csv are guaranteed to be on LOD." << std::endl;
+		  }
+		  else if (time_to_coalescence > 0) {
+			  std::cout << "Time to Coalescence for the last organism written to LOD_data.csv (at time " << dataSequence[data_seq_index - 1] << ") is "
+				  << time_to_coalescence << "." << std::endl;
+		  }
+		  else if (time_to_coalescence == 0) {
+			  std::cout << "The last organism written to LOD_data.csv (at time " << dataSequence[data_seq_index - 1] << ") had a time to coalescence of 0.\n" <<
+				  "Rejoice! This organism is on LOD!" << std::endl;
+		  }
+		  else {
+			  std::cout << "The last organism written to LOD_data.csv had a negitive time to coalescence.\n" <<
```

Comment:
If this is a true error, then we should exit(1), instead of a clean exit.

Reply:
I agree that removing the clause or making a very visible error message is good.